boot/modeenv.go

```diff
@@ -69,6 +87,8 @@ func (m *Modeenv) Unset() bool {
 	return !m.read
 }
 
+// Write outputs the modeenv to the file specified by
+// "rootdir/dirs.SnapModeenvFile".
```

Comment:
... file at `<rootdir>/var/lib/snapd/modeevn`.

Reply:
This one too